resources/views/filament/components/server-console.blade.php

```diff
@@ -142,12 +153,6 @@ class="w-full focus:outline-none focus:ring-0 border-none"
                         'args': [null]
                     }));
                     break;
-                case 'install started':
-                    $wire.dispatch('console-install-started');
-                    break;
-                case 'install completed':
-                    $wire.dispatch('console-install-completed');
-                    break;
```

Comment:
Why are these being removed?

Reply:
Because the listeners have been removed.

Currently the alert banners don't support refreshing/ updating. But that's fine for now since you have to F5 reload the page anyways after a install. (because of the navigation menu)